so-elastalert/files/create_index.py

```diff
@@ -193,14 +178,13 @@ def main():
         client_key = data.get('client_key')
         index = args.index if args.index is not None else data.get('writeback_index')
         old_index = args.old_index if args.old_index is not None else None
-        index_settings = data.get('index_settings')
     else:
         username = args.username if args.username else None
         password = args.password if args.password else None
         bearer = args.bearer if args.bearer else None
         api_key = args.api_key if args.api_key else None
         aws_region = args.aws_region
-        host = args.host if args.host else input('Enter Elasticsearch host: ')
+        host = args.hosts if args.host else input('Enter Elasticsearch host: ')
```

Comment:
Why is this `args.hosts` (plural), yet it's using it only if `args.host` (singular) is defined?